docs/learn/learn-async-backing.md

```diff
@@ -0,0 +1,116 @@
+instruction in "traditional" architectures, where some instructions may be executed before others
+are complete. Instructions may also be executed in parallel, enabling multiple parts of the
+processor to be working on potentially different instructions at the same time.
+
+Bundles of state transitions represented as blocks may be processed similarly. In the context of
+Polkadot, it aims to overall increase the throughput of the entire network.
+
+:::info
+
+In order to realize which exact processes this upgrade improves, it is recommended to be familiar
+with the stages of parablock validation, which you may [read here](./learn-parachains-protocol.md).
+
+:::
+
+Asynchronous backing enables logical pipelining over the parablock 
+[**generation**](https://wiki.polkadot.network/docs/learn-parachains-protocol#collators),
+[**backing**](https://wiki.polkadot.network/docs/learn-parachains-protocol#parachain-phase), and
+[**inclusion**](https://wiki.polkadot.network/docs/learn-parachains-protocol#inclusion-pipeline)
+processes. A parablock may be at different stages, but multiple blocks should be able to process
+simultaneously (in parallel) if needed. Most notably, parablock N + 1 can be generated and backed during the same time in which its predecessor, parablock N, is undergoing inclusion on the relay chain.
+processes can occur while their ancestors are included on the relay chain.
+
+This pipeline will allow [collators](./learn-parachains-protocol.md#collators) to include an
+estimated ~3-5x more transactions/data while speeding up parachain block times from 12 to 6 seconds.
+In short, Polkadot with asynchronous backing will deliver an estimated ~6-10x more blockspace to its parachains.
+to Polkadot parachains.
+
+Asynchronous Backing has three overarching goals:
+
+1. Decrease parachain block (parablock) validation time to **6 seconds** from **12 seconds**
+2. Increase the amount of usable blockspace as per above, meaning more state changes are allowed per
+   relay chain block.
+3. Allow for parablocks to be re-proposed to the network if they are not included successfully on the
+   first attempt.
+
+Asynchronous backing provides a form of **contextual execution**, which allows for more time for
+parachain collators to fit more transactions and ready block candidates for backing and inclusion.
+**Contextual execution** refers to how a parablock can begin being built earlier using the context
+provided by an _unincluded segment_ of recent block ancestors.
+
+**Unincluded segments** are chains of candidate blocks that are not yet included in the relay chain.
+Parablocks can be added to this unincluded segment and no longer have to wait for the latest
+included parent block of the relay chain. The core functionality that asynchronous backing brings is
+the ability to build on these unincluded segments of block ancestors rather than ancestors included in the relay
+chain state.
```

Comment:
```suggestion
**Unincluded segments** are chains of candidate blocks that are not yet included in the relay chain.
Parablocks can be added to this unincluded segment without waiting for the latest
included parent block of the relay chain. The core functionality that asynchronous backing brings is
the ability to build on these unincluded segments of block ancestors rather than ancestors included in the relay
chain state.
```